plugin/plugin_test.go

```diff
@@ -270,6 +283,8 @@ func TestContainerdPlugin(t *testing.T) {
 		"io.containerd.internal.v1.opt",
 		"io.containerd.grpc.v1.cri",
 		"io.containerd.internal.v1.restart",
+		"io.containerd.tracing.processor.v1.otlp",
+		"io.containerd.internal.v1.tracing",
 		"io.containerd.grpc.v1.introspection",
```

Comment:
Curious why `io.containerd.grpc.v1.introspection` is registered at the end? Does it require otlp plugins to be registered before it?

Reply:
From [code](https://github.com/containerd/containerd/blob/main/plugin/plugin_test.go#L174) `io.containerd.grpc.v1.introspection` does require all other plugins registered. I didn't dig deep to see what does that plugin do though.